src/api/submission/protocol_validations.py

```diff
@@ -113,7 +116,9 @@ def validate(self):
 
 class SampleUnitValidation(ProtocolValidation):
     DEPTH_RANGE = (1, 30)
+    DATE_TIME_RANGE = (datetime.time(6,0), datetime.time(19,0))
     DEPTH_MSG = str(_("Depth value outside range of {} and {}".format(*DEPTH_RANGE)))
+    DATE_TIME_MSG = str(_("Sample time outside of range {} and {}".format(*DATE_TIME_RANGE)))
```

Comment:
Do we need to wrap the `_()` with `str()` for all these?

Reply:
str() forces it to be evaluated right away rather than lazy load